musi/pk.srt

```diff
@@ -4556,35 +4556,48 @@ point. let it open then.
 
 980
 01:49:30,533 --> 01:49:43,577
-gazing you, bearing you,
+mi lukin e sina. mi awen e sina.
+mi pali ala e ijo ante li awen lon monsi sina.
+==gazing you, bearing you,
 i forgotton everything and just follow you.
 
 981
 01:49:43,596 --> 01:49:49,724
-this is such a waste of time,
+ni li weka tenpo a.
+olin li weka tenpo.
+==this is such a waste of time,
 love is a waste of time.
 
 982
 01:49:56,279 --> 01:50:09,089
-then also i decided atleast once in this life,
+ni la tenpo wan pi lon mi anu tenpo mute la 
+mi weka e tenpo mi. olin li weka tenpo.
+==then also i decided atleast once in this life,
 i will waste my time. love is a waste of time.
 
 983
 01:50:32,436 --> 01:50:43,378
-many time i see mirror and do my hair.
+tenpo mute la mi lukin e mi li pona e linja mi.

```

Comment:
(One cannot put a second li in a sentence with mi as subject?) pona. mi sona ala e ni. It also seems to fit better the song. In my subtitles (in Portuguese) there is one sentence for each action he does many times.